src/cli.spec.js

```diff
@@ -44,7 +44,10 @@ describe('cli', () => {
 
       expect(projectInfos.getProjectInfos).toHaveBeenCalledTimes(1)
       expect(cli.askQuestions).toHaveBeenCalledTimes(1)
-      expect(cli.askQuestions).toHaveBeenCalledWith(projectInformations)
+      expect(cli.askQuestions).toHaveBeenCalledWith(
+        projectInformations,
+        undefined
```

Comment:
Could you add a new test with `yes` options that check `inquirer` is not called and default values are used to generated the `answersContext`?